includes/template-library/sources/remote.php

```diff
@@ -210,6 +216,91 @@ public function get_data( array $args, $context = 'display' ) {
 		return $data;
 	}
 
+	/**
+	 * Get templates data from transient or by remote request.
+	 * The remote request will happen in one of those 3 conditions:
+	 * 1. Force update.
+	 * 2. The data saved in the transient is empty.
+	 * 3. The container feature has changed since the last data fetch.
+	 *
+	 * @param $force_update
+	 * @return array
+	 */
+	private function get_templates_data( $force_update ) {
+		$templates_data_cache_key = static::TEMPLATES_DATA_TRANSIENT_KEY_PREFIX . ELEMENTOR_VERSION;
+		$templates_data_layout_type_cache_key = static::TEMPLATES_DATA_LAYOUT_TYPE_TRANSIENT_KEY;
```

Comment:
Why create another variable if it's the same as the const?

Reply:
Removed